test/README.md

```diff
@@ -99,3 +99,15 @@ get backtrace from).
 approximately 90% of the tests. Should be used when finding cores is too hard 
 for some reason. Normally (on OS X or when `NVIM_TEST_CORE_GLOB_DIRECTORY` is 
 defined and this variable is not) cores are checked for after each test.
+
+`NVIM_TEST_RUN_TESTTEST` (U) (1): allows running `test/unit/testtest_spec.lua` 
+used to check how testing infrastructure works.
+
+`NVIM_TEST_NO_TRACE` (U) (1): omits getting traces from tests. This means that 
+if tests crashed without core dump you will have no clues regarding where, but 
+this makes tests run a lot faster. Combine with `NVIM_TEST_MAIN_CDEFS` for 
+maximal speed.
+
+`NVIM_TEST_TRACE_EVERYTHING` (U) (1): by default unit test only record C calls 
+which is faster then recording everything. Set this variable to 1 if you want to 
+see all traces.
```

Comment:
To mitigate proliferation of env vars, can we combine NVIM_TEST_NO_TRACE and NVIM_TEST_TRACE_EVERYTHING into a pseudo-ternary:

```
NVIM_TEST_TRACE_LEVEL = 0 # no trace
NVIM_TEST_TRACE_LEVEL = 1 # some
NVIM_TEST_TRACE_LEVEL = 2 # everything
```